bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2478,7 +2478,7 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( IsTemplatedSTLClass( name, "vector" ) ) {
+   else if ( IsTemplatedSTLClass( name, "vector" ) || IsTemplatedSTLClass( name, "ROOT::VecOps::RVec" )) {
 
       if ( HasAttrDirect( pyclass, PyStrings::gLen ) && HasAttrDirect( pyclass, PyStrings::gAt ) ) {
```

Comment:
Are we sure we want everything that is enclosed by this if to be executed for `RVec` as well?

Reply:
I think yes.